blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedMultiIngestionHandler.java

```diff
@@ -139,16 +139,14 @@ public void handle(ChannelHandlerContext ctx, FullHttpRequest request) {
             }
         } catch (JsonParseException ex) {
             log.debug(String.format("BAD JSON: %s", body));
-            log.error(ex.getMessage(), ex);
             DefaultHandler.sendErrorResponse(ctx, request, ex.getMessage(), HttpResponseStatus.BAD_REQUEST);
         } catch (InvalidDataException ex) {
             log.debug(String.format("Invalid request body: %s", body));
-            log.error(ex.getMessage(), ex);
             DefaultHandler.sendErrorResponse(ctx, request, ex.getMessage(), HttpResponseStatus.BAD_REQUEST);
         } catch (TimeoutException ex) {
             DefaultHandler.sendErrorResponse(ctx, request, "Timed out persisting metrics", HttpResponseStatus.ACCEPTED);
         } catch (Exception ex) {
-            log.debug(String.format("BAD JSON: %s", body));
+            log.debug(String.format("Exception processing: %s", body));
```

Comment:
If we get some exception which we are not expecting, I think we need some visibility to it. Should we log error here?